### PR TITLE
Juanstevan1/feature/user management

### PR DESCRIPTION
**Pull request by Juanstevan1**

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        $user->save();
+    
+        return redirect()->route('user.index')->with('success', __('User created successfully.'));
+    }
+
+    public function show(int $id): View
+    {
+    $viewData = [];
+    $viewData['user'] = User::findOrFail($id);
+    $viewData['title'] = __('Detalles del Usuario');
```
Comment (alejoriosm04) [inline app/Http/Controllers/UserController.php]: Do not combine English with Spanish. @Juanstevan1

File: app/Models/User.php [hunk 2]
```diff
@@ -2,47 +2,143 @@
...
+            'image' => 'nullable|image|mimes:jpg,jpeg,png,gif|max:2048', 
+        ]);
+    }
+    
+    public function favList(): BelongsToMany
+    {
+        return $this->belongsToMany(Product::class, 'user_favorites_products', 'user_id', 'product_id');
+    }
+
+    public function pets()
```
Comment (alejoriosm04) [inline app/Models/User.php]: Specify return type. @Juanstevan1

**Review by alejoriosm04 — COMMENTED** [review]
- Execute Laravel Pint.
- Do not combine English with Spanish in the Lang, mainly in the `Views`.

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (Juanstevan1) [reply app/Http/Controllers/UserController.php]: Done

File: app/Models/User.php (hunk 2 above)
Comment (Juanstevan1) [reply app/Models/User.php]: Done

**Review by alejoriosm04 — COMMENTED** [review]
Lang is still not corrected. @Juanstevan1

**Pull request merged**